.github/workflows/release.yaml

```diff
@@ -17,9 +17,19 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.9
+    - name: Install apt dependencies
+      run: |
+        sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
+        sudo apt-get update
+        sudo apt-get install gdal-bin libgdal-dev
     - name: Install pip dependencies
-      run: pip install .[tests]
+      run: |
+        pip install gdal tqdm  # TODO: these deps shouldn't be needed
```

Comment:
@isaaccorley the indices tutorial imports things that aren't TorchGeo deps, does it need to?

Reply:
I can revamp them but requiring that notebooks only use torchgeo deps might be too restrictive.

Reply:
The code definitely uses these two libraries.
- tqdm is simple to include as a dependency, it will be indirectly included anyway, and is quite useful in notebooks and command line scripts
- The gdal bit can be replaced quite easily with rasterio

Reply:
Agreed, let's do this for the next release